modules/release-notes/con-relnotes-notable-features.adoc

```diff
@@ -5,25 +5,25 @@ This section highlights new features in {product} {product-version}.
 
 == {product} Operator is now generally available (GA)
 
-You can use the {product} Operator to install {product-short} on your {ocp-short}cluster. For more information, see the link:{LinkAdminGuide}#proc-install-rhdh-ocp-operator_admin-rhdh[Installing {product} using the Operator] section in the _Administration guide_.
+You can use the {product} Operator to install {product-short} on your {ocp-short}cluster. For more information, see the link:{LinkAdminGuide}#proc-install-rhdh-ocp-operator_{context}[Installing {product} using the Operator] section in the _Administration guide_.
```

Comment:
this item was new in 1.1. Not sure why it's still here in 1.2. Can you remove it?